bindata/assets/secondary-scheduler/deployment.yaml

```diff
@@ -23,6 +23,9 @@ spec:
       containers:
         - name: "secondary-scheduler"
           image: ${IMAGE}
+          env:
+          - name: EnableOpenShiftAuth
```

Comment:
Based on https://github.com/paypal/load-watcher/pull/49/files#diff-7aef12efe02ab3844f2b733db92ff48f984064475bb78514a3f25556fb516fc2R46 should not the name be `ENABLE_OPENSHIFT_AUTH`?

Reply:
@ingvagabund , yes, you are right. I almost forgot that! Thanks.